lightning/src/blinded_path/utils.rs

```diff
@@ -177,3 +177,26 @@ impl Writeable for Padding {
 		Ok(())
 	}
 }
+
+
+/// A wrapper struct that stores the largest packet size for the given [`BlindedPath`].
+/// This helps us calculate the appropriate padding size for the tlvs when writing them.
+pub struct WithPadding<T: Writeable> {
+	/// Length of the packet with the largest size in the [`BlindedPath`].
+	pub(crate) max_length: usize,
+	/// The current packet's TLVs.
```

Comment:
"Current" implies some context, but the docs should be read independent of where this is used. How about "TLVs for a packet to write"?

Reply:
Sounds good!
Updated it to "_TLVs for a packet to be written_" in [pr3177.07](https://github.com/shaavan/rust-lightning/commits/pr3177.07)

Thanks for the suggestion!